core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -199,7 +199,8 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
       runtimeOptions,
       useCallCache,
       billingProject.cromwellBackend.getOrElse(defaultBackend),
-      workflowFailureMode
+      workflowFailureMode,
+      google_labels = Map("terra-submission-id" -> s"terra-${submissionId.toString}")
```

Comment:
perhaps a nit, but do we want to use `terra` here?  There's Terra UI, and then the whole analysis platform backend services suite, which I'm sure some people call Terra, but others don't.   Like, when AoU starts submitting workflows, do we want their submissions called `terra-submissions` still?  Or might there be value in keeping the key something generic like, `submission-id`, and the value something that can have a namespacing prefix, like you have `terra-`

Reply:
oh, sorry -- i just hit merge and the page refreshed and i saw this. my understanding is that everything is called terra now. it's certainly a case of "firecloud, powered by terra"; i'm not sure if that's the ultimate fate of AoU also.

secondly, this ending up on PAPI VMs and our bigquery cost export, so i don't really think end-users will be seeing this anyway

Reply:
No worries.  I was totally fine with the PR, hence the approval, was just thinking about the labels in a potentially inconsequential way, so just wanted to ask.  It's all fine with me.